CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+1.2.4 (2022-02-10)
```

Comment:
This is a breaking change for anyone running on python 3.6.
Can you instead bump to 2.0.0? Also add in the changes dropping support for py3.6?